desktop/conf/pseudo-distributed.ini.tmpl

```diff
@@ -368,6 +368,10 @@
     # The service to use when querying PAM.
     ## pam_service=login
 
+    # To use python unix pwd module to get the username from the entered credentials in hue if Centrify like pam service is in use.
+    # This will set the username to what is being returned by the pwd module.
+    ## use_pam_pwd_module=false
```

Comment:
will make this change.  I did the normal pam testing with this and it worked fine... The main thing is for our internal customer to test this out since we don't have centrify to do the name conversion ;)